### PR TITLE
Remove duplicate test block creation methods

### PR DESCRIPTION
**Pull request by haozhun**
A simple test-only change

### CODE REVIEW

**Comment by cberner** [comment]
There are a lot of test failures

**Comment by haozhun** [comment]
Fixed test failures. The original change ran into issues with the details of how Java resolves function calls.

File: presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestCorrelationAggregation.java [hunk 1]
```diff
@@ -76,6 +77,11 @@ private void testNonTrivialAggregation(double[] y, double[] x)
         PearsonsCorrelation corr = new PearsonsCorrelation();
```
Comment (cberner) [inline presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestCorrelationAggregation.java]: It looks like this helper is only used in one place, so I would just change it to take Double[] instead. Also, you could just inline the whole helper

**Comment by cberner** [comment]
Looks good

File: presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestCorrelationAggregation.java (hunk 1 above)
Comment (haozhun) [inline presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestCorrelationAggregation.java]: double[] is required for arguments of `corr.correlation`

**Pull request merged**